components/server/ee/src/user/user-service.ts

```diff
@@ -78,7 +85,8 @@ export class UserServiceEE extends UserService {
 
         // 1. check the license
         const userCount = await this.userDb.getUserCount(true);
-        if (!this.licenseEvaluator.hasEnoughSeats(userCount)) {
+        if (userCount > -1) {
+            //!this.licenseEvaluator.hasEnoughSeats(userCount)) {
             const msg = `Maximum number of users permitted by the license exceeded`;
```

Comment:
**suggestion**: Adding one more alternative copy with a different copy structure, but we can work on improving the error messages in a later iteration. Cc @lucasvaltl 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-08-01 at 9 46 00 PM" src="https://user-images.githubusercontent.com/120486/182221588-d10d1293-6c6b-4607-8e1d-a2125428966a.png"> | <img width="1440" alt="Screenshot 2022-08-01 at 9 45 39 PM" src="https://user-images.githubusercontent.com/120486/182221598-2024355b-a8c6-432c-b11f-36842752677a.png"> |

Reply:
I'm ok with this in general, but the terminology "instance user cap" seems off to me. We don't use it anywhere else, and we usually talk about seats or just users. I'd suggest we go with "**Maximum number of users reached.** We could not sign you in because this instance has reached the maximum number of allowed users."

Reply:
Good point, @lucasvaltl ! Agree, let's go with your updated suggestion:

> **Maximum number of users reached.** We could not sign you in because this instance has reached the maximum number of allowed users."

Cc @geropl